src/components/Container/Container.tsx

```diff
@@ -1,33 +1,44 @@
-import { StyleProp, ViewStyle } from 'react-native';
-import { View, ViewProps } from '@bluebase/components';
+import { SCREEN_SIZE, View, ViewProps } from '@bluebase/components';
+
 import { MAX_CONTAINER_WIDTH } from '../../constants';
 import React from 'react';
-import { ScreenSizeConsumer } from '../ScreenSize';
-import { Theme } from '@bluebase/core';
+import { useLayout } from '@react-native-community/hooks';
+import { useScreenSize } from '../../hooks';
+
+export interface ContainerStyles {}
 
-export interface ContainerStyles {
-	paddingHorizontalXS: StyleProp<ViewStyle>;
+export interface ContainerProps extends ViewProps {
+	dimension?: 'layout' | 'screen';
+	maxSize?: SCREEN_SIZE;
+	styles: any;
```

Comment:
should not be any. discuss with @adnan1naeem

Reply:
done